torchbenchmark/operators/flash_attention/operator.py

```diff
@@ -109,6 +110,7 @@ class Operator(BenchmarkOperator):
 
     def __init__(self, tb_args: argparse.Namespace, extra_args: Optional[List[str]] = None):
         super().__init__(tb_args, extra_args)
+        self.use_cuda_graphs = False
```

Comment:
Currently TMA doesn't work with cuda_graph, I wonder if there is a way of disabling it only for one variant triton_tutorial_flash_v2_tma. CC @xuzhao9

Reply:
Possibly dumb question since I haven't read the code: is there any way we can set up the TMA descriptors outside the benchmarked function? That would make cudagraphs possible. cc @bertmaher who did something similar recently.

Reply:
@manman-ren For fair comparison, I would actually prefer to keep the same function across all kernels. If `triton_tutorial_flash_v2_tma` is not compatible with `do_bench_cudagraph()`, let's just use `do_bench()` across all kernels.

Reply:
Yep, I did some tricks in fp8_gemm to allocate the TMA descriptors outside of the benchmarked region, let me put up a suggestion for how to do that here

Reply:
Ok, I think you want something like this: https://gist.github.com/bertmaher/971cbae202d314913594a97e45058553

So when you're setting up the benchmark, you first call `alloc_tma_desc` to create the descriptors for k, v, then you use those in the benchmark function rather than reallocating

Is it beneficial to use TMA for q and o as well as just k v?

Reply:
Thanks Bert! I don't see perf win for TMA with this implementation on a few cases I checked. I will try to cover more cases and report the results here.

I am starting with k and v since they are handled inside the loop. I am thinking about landing the initial implementation and land additional improvements as separate patches. What do you think? @bertmaher @embg